inst/tests/test-tags.r

```diff
@@ -570,6 +570,32 @@ test_that("Indenting can be controlled/suppressed", {
   )
 })
 
+test_that("cssList tests", {
+  expect_identical("", css())
+  expect_identical("", css())
+  expect_identical(
+    css(
+      font.family = 'Helvetica, "Segoe UI"',
```

Comment:
If someone specifies `c('Helvetica','Segoe UI')`, should `css` know that certain values need quoting?